dotnet/src/SemanticKernel.UnitTests/TemplateEngine/Blocks/CodeBlockTests.cs

```diff
@@ -56,11 +53,8 @@ public async Task ItThrowsIfAFunctionCallThrowsAsync()
         this._skills.Setup(x => x.GetFunction("functionName")).Returns(function.Object);
         var target = new CodeBlock("functionName", this._logger.Object);
 
-        // Act
-        var exception = await Assert.ThrowsAsync<TemplateException>(async () => await target.RenderCodeAsync(context));
-
-        // Assert
-        Assert.Equal(TemplateException.ErrorCodes.RuntimeError, exception.ErrorCode);
+        // Act & Assert
+        await Assert.ThrowsAsync<SKException>(async () => await target.RenderCodeAsync(context));
```

Comment:
Not sure if this test method should be a `Task`, the others aren't. I would move this one to be sync or the others to be async

Reply:
This test is for the `target.RenderCodeAsync` method, which works asynchronously and should be awaited. The test method should return a `Task` and be marked as `async` for XUnit to run and await it. While we could use the `Wait()` method to make the test method synchronous and wait for `RenderCodeAsync` to complete, why would we do that?